lib/views/after_auth_screens/profile/profile_page.dart

```diff
@@ -94,154 +94,144 @@ class ProfilePage extends StatelessWidget {
-                          mainAxisAlignment: MainAxisAlignment.start,
-                          children: [
-                            SizedBox(
-                              height: SizeConfig.screenHeight! * 0.05,
-                            ),
-                            CustomListTile(
-                              key: homeModel!.keySPAppSetting,
-                              index: 0,
-                              type: TileType.option,
-                              option: Options(
-                                icon: const Icon(
-                                  Icons.phonelink_setup,
-                                  size: 30,
-                                ),
-                                title: AppLocalizations.of(context)!
-                                    .strictTranslate('App Settings'),
-                                subtitle:
-                                    '${AppLocalizations.of(context)!.strictTranslate("Language")}, ${AppLocalizations.of(context)!.strictTranslate("dark mode")}, ${AppLocalizations.of(context)!.strictTranslate("font size")}',
-                              ),
-                              onTapOption: () {
-                                navigationService
-                                    .pushScreen("/appSettingsPage");
-                              },
-                            ),
-                            SizedBox(
-                              height: SizeConfig.screenHeight! * 0.05,
-                            ),
-                            CustomListTile(
-                              key: const Key('TasksByUser'),
-                              index: 1,
-                              type: TileType.option,
-                              option: Options(
-                                icon: const Icon(
-                                  Icons.task_outlined,
-                                  size: 30,
-                                ),
-                                title: AppLocalizations.of(context)!
-                                    .strictTranslate('My Tasks'),
-                                subtitle: AppLocalizations.of(context)!
-                                    .strictTranslate(
-                                        "View and edit all tasks created by you"),
+                      Column(
+                        mainAxisAlignment: MainAxisAlignment.start,
+                        children: [
+                          SizedBox(
+                            height: SizeConfig.screenHeight! * 0.05,
+                          ),
+                          CustomListTile(
+                            key: homeModel!.keySPAppSetting,
+                            index: 0,
+                            type: TileType.option,
+                            option: Options(
+                              icon: const Icon(
+                                Icons.phonelink_setup,
+                                size: 30,
```

Comment:
Wrap all the items with Flutter's `Expanded` Widget except the last one `from PALISADOES` to the bottom to ensure that it always sticks to the bottom regardless of the screen size.
More : https://api.flutter.dev/flutter/widgets/Expanded-class.html

Reply:
@Tanmoy741127, Can you wrap the items with flutter's `Expanded` widget as Siddhesh recommended?

Reply:
@JamaicanFriedChicken sure. I have actually missed the comment. I will do that and update .

Reply:
@SiddheshKukade @JamaicanFriedChicken 
has wrapped the children in expanded in latest commit